kata-deploy/kata-cleanup.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-cri-o-kata-cleanup
+  namespace: kube-system
+spec:
+  selector:
+      matchLabels:
+        name: kubelet-cri-o-kata-cleanup
+  template:
+    metadata:
+        labels:
+          name: kubelet-cri-o-kata-cleanup
+    spec:
+      serviceAccountName: kata-label-node
+      nodeSelector:
+          container-runtime: cri-o
+          kata-runtime: cleanup
+      containers:
+      - name: kube-kata-cleanup
+        image: egernst/kata-deploy
+        imagePullPolicy: Always
+        command: [ "sh", "-c" ]
+        args:
+        - kubectl label node $NODE_NAME container-runtime- kata-runtime-;
```

Comment:
should we use `katacontainers.io/<label-name>` ?

Reply:
yeah... probably a good idea.

Reply:
done